Platform/OpenCanopy/BitmapFont.c

```diff
@@ -586,26 +602,55 @@ BmfGetTextInfo (
   return TextInfo;
 }
 
+STATIC
+EFI_GRAPHICS_OUTPUT_BLT_PIXEL
+mBlack = { 0, 0, 0, 255 };
+
+STATIC
+EFI_GRAPHICS_OUTPUT_BLT_PIXEL
+mWhite = { 255, 255, 255, 255 };
+
 STATIC
 VOID
-CopyInverted (
+BlendMem (
   EFI_GRAPHICS_OUTPUT_BLT_PIXEL *Dst,
-  EFI_GRAPHICS_OUTPUT_BLT_PIXEL *Src,
+  EFI_GRAPHICS_OUTPUT_BLT_PIXEL *AlphaSrc,
+  EFI_GRAPHICS_OUTPUT_BLT_PIXEL *Color,
   UINTN                         PixelCount
   )
 {
   UINTN  Index;
 
   for (Index = 0; Index < PixelCount; ++Index) {
-    Dst->Red       = 255 - Src->Red;
-    Dst->Green     = 255 - Src->Green;
-    Dst->Blue      = 255 - Src->Blue;
-    Dst->Reserved  = Src->Reserved;
+    // We assume that the font is generated by dpFontBaker
```

Comment:
Please use surrounding `//` for upper and lower lines for comments.

Reply:
Maybe we should implement some option in OcPngLib to allow loading gray-only images as alpha-only, so it would set its `Reserved` component instead of `Blue`, `Green`, `Red`?

Reply:
I do not mind, but I am not sure how to do it best. What would you like to change it to in more detail?

Reply:
No, let's leave as is, because it would involve changes to lodepng, not our code.